CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contributing
```

Comment:
Yeah, I forgot to do that before telling you of the changes. It should link to it now